_posts/elements/2015-03-25-headings.md

```diff
@@ -20,78 +20,78 @@ info: Headings are used to define important topics and section content into dige
 </div>
 
 <div class="example-set">
-    <h4 class="hd-4">Crying in the Courage</h4>
-    <p class="copy copy-base">And gosh and jovial this much but angelfish after and <a href="#">far but inside and this roadrunner</a> that this oh when yikes therefore far amidst goldfinch darn lazily inside a abashed hello so after nutria gosh much and mallard out shoddily sang more the genial close destructively on then as less.</p>
+    <h4 class="hd-4">Crying in the Courage (h4)</h4>
+    <p class="copy copy-base">And gosh and jovial this much but angelfish after and <a href="#">far but inside and this roadrunner</a> that this oh when yikes therefore far amidst goldfinch darn lazily inside a abashed hello so after nutria gosh much and mallard out shoddily sang more the genial close destructively on then as less. (base copy)</p>
 
-    <h4 class="hd-4 emphasized">Lovely Year</h4>
-    <p class="copy copy-base">Wobbled this darn and warthog compatibly that smoothly tediously ashamed while grouped around tangibly inanimately unkindly wailed awkward together like angelfish fastidious as lingeringly among that fitted crud cried orca strove on and hello far kissed octopus regarding confused less one and the oh.</p>
+    <h4 class="hd-4 emphasized">Lovely Year (emphasized h4)</h4>
+    <p class="copy copy-base">Wobbled this darn and warthog compatibly that smoothly tediously ashamed while grouped around tangibly inanimately unkindly wailed awkward together like angelfish fastidious as lingeringly among that fitted crud cried orca strove on and hello far kissed octopus regarding confused less one and the oh. (base copy)</p>
 
-    <h4 class="hd-4 de-emphasized">The Fallen Legacy</h4>
-    <p class="copy copy-base">Lackadaisical fallibly next annoyingly shrugged and copiously inexhaustibly rode however peculiar more this spuriously flagrantly vigilantly crud more woolly effectively macaw among cardinal more alas darn near a conic primly more truly tarantula hello inventoried.</p>
+    <h4 class="hd-4 de-emphasized">The Fallen Legacy (de-emphasized h4)</h4>
+    <p class="copy copy-base">Lackadaisical fallibly next annoyingly shrugged and copiously inexhaustibly rode however peculiar more this spuriously flagrantly vigilantly crud more woolly effectively macaw among cardinal more alas darn near a conic primly more truly tarantula hello inventoried. (base copy)</p>
 </div>
 
 <div class="example-set">
-    <h5 class="hd-5">Diamond World</h5>
+    <h5 class="hd-5">Diamond World (h5)</h5>
     <p>Behind the hello far that that alas weasel vitally went one properly darn ladybug crud agonizing gosh so wicked goodness in until hedgehog some far close jeez some this hired dachshund bawdy frog outdid tedious after over but <a href="#">nonchalant ouch jeez thus</a> more some until capitally hen outside tarantula.</p>
```

Comment:
Q: It looks like this example-set has no classes for its paragraphs. Is this intentional or should I add a specific class to the paragraphs here? (impacts entire example set)